src/components/BottomNavigation/BottomNavigation.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from "next/image";
+import { Avatar, AvatarFallback, AvatarImage } from "@radix-ui/react-avatar";
```

Comment:
Use Avatar component from shadcn

Reply:
Thanks, I installed it, but auto-import imported other component :D